applications/callflow/src/module/cf_directory.erl

```diff
@@ -79,6 +79,7 @@
 
 -define(PROMPT_ENTER_PERSON_LASTNAME, <<"dir-enter_person_lastname">>). %% Please enter the first few letters of the person's lastname
 -define(PROMPT_ENTER_PERSON_FIRSTNAME, <<"dir-enter_person_firstname">>). %% Please enter the first few letters of the person's firstname
+-define(PROMPT_ENTER_PERSON_NAME, <<"dir-enter_person_name">>). %% Please enter the first few letters of the person's name
```

Comment:
Everything looks good except for this prompt? Maybe you meant this one? https://github.com/2600hz/kazoo-sounds/blob/master/kazoo-core/prompts.txt#L46

Reply:
Yes, you're right. I was still using our own repo for kazoo-sounds when testing. I'll fix that

Reply:
Hi James. I'm adding a prompt for dir-enter_person_name in kazoo sounds to support this change (PR coming). We only have this file in English. Is there a need to support the prompt in other languages?

Reply:
@jamesaimonetti see https://github.com/2600hz/kazoo-sounds/pull/7